stdlib/bigarray.mli

```diff
@@ -186,7 +186,12 @@ val int16_unsigned : (int, int16_unsigned_elt) kind
 (** See {!Bigarray.char}. *)
 
 val int : (int, int_elt) kind
-(** See {!Bigarray.char}. *)
+(** See {!Bigarray.char} and {!section:elementkinds}.
+
+ OCaml integers (signed, 31 bits on 32-bit architectures,
+   63 bits on 64-bit architectures).
+ They do not match the C [int] type.
```

Comment:
Let be more explicit: since we are writing down a warning, it should looks like a warning even for people which do not have the context of this PR.

> Beware that this a bigarray containing OCaml integers (signed, 31 bits on 32-bit architectures,
   63 bits on 64-bit architectures), which does not match the [C] int  type.